backend/src/main/kotlin/com/group7/artshare/repository/RegisteredUserRepository.kt

```diff
@@ -3,8 +3,8 @@ package com.group7.artshare.repository
 import com.group7.artshare.entity.RegisteredUser
 import org.springframework.data.jpa.repository.JpaRepository
 import org.springframework.stereotype.Repository
-import java.util.*
 
 @Repository
 interface RegisteredUserRepository : JpaRepository<RegisteredUser, Long> {
+    fun findByAccountInfo_Username(username : String) : RegisteredUser?
```

Comment:
We already have a method for this in RegisteredUserService. 
You should use that one and remove this one. 
Probably this won't work with the Artist users.

Reply:
I tested, it works with Artist users. But I will switch to your method for now since I see it has some usages already. But lets keep in mind there is a way of accessing this kind of linked classes with such syntax for future necessities.